spec/fixtures/teachers.yml

```diff
@@ -32,16 +30,14 @@ long:
   status: 1
   school: berkeley
   more_info: ''
-  validated: false
-  denied: false
+  application_status: Pending
   school_id: 1
   education_level: 2
 
 tony:
   first_name: 'Tony'
   last_name: 'Stark'
-  validated: true
-  denied: false
+  application_status: Validated
```

Comment:
I'm not sure about this, but do we need to put quotes here? Names seem to have quotes around

Reply:
As per [this](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) it seems like quotes aren't needed for most strings unless there are special characters. In the fixtures it seems like some names are quoted and some aren't; for consistency's sake I can change this to unquote all the normal strings and quote the emails (because of the @)